content/contribute.md

```diff
@@ -0,0 +1,62 @@
+---
+draft: false
+title: Contribute to the MaxMind Community
+---
+
+We welcome the efforts of our community of users to develop robust, unofficial
+tools for use with our products and services. One of the reasons that we keep
+our APIs open is to encourage development and collaboration as you put our data
+to use.
+
+## Getting Started
+
+Before you begin developing tools for use across the community, you should fully
+familiarize yourself with the relevant official APIs and specifications:
+
+- [minFraud service REST API](/minfraud/api-documentation/#request-and-response-api-references)
+- [GeoIP2 and GeoLite2 service REST API](/geoip/docs/web-services#request-and-response-api-references)
+- [GeoIP2 and GeoLite2 MMDB format specification](https://github.com/maxmind/MaxMind-DB/blob/main/MaxMind-DB-spec.md)
+
+You may also want to look at some of the unofficial APIs that we’ve featured.
+While MaxMind does not support or maintain unofficial APIs, you can get a
+feeling for the quality of contributions that we feature:
+
+- [minFraud service Magento 2 Plugin](https://www.weltpixel.com/magento2-maxmind-fraud-prevention-minfraud.html)
+- [GeoIP2 and GeoLite2 web services Unofficial APIs](/geoip/docs/web-services/#third-party-client-apis)
+- [GeoIP2 and GeoLite2 database Unofficial APIs](/geoip/docs/databases/#unofficial-client-apis)
+- [GeoIP2 and GeoLite2 database Integrations (for MMDB format)](/geoip/docs/databases/#integrations)
+
+You can also peruse our organization and projects on GitHub to get a sense of
+how we develop for MaxMind’s products and services. Some good examples to look
+at are:
+
+- [Ruby MMDB Reader](https://github.com/maxmind/MaxMind-DB-Reader-ruby)
+- [minFraud .NET API](https://github.com/maxmind/minfraud-api-dotnet)
+
+## What Tools are Useful to Build
+
+Well-developed client APIs in languages we don’t already support are especially
+useful. We also appreciate efforts to develop plugins for popular content
+management systems, as well as analytics and shopping cart software. In order
+for your tools to be featured on our developer portal, they must work with one
+of our paid products or services (GeoIP2 or minFraud). GeoLite2 databases and
+service integrations should be compatible with GeoIP2 products and services with
+minor modifications.
+
+## Sharing Your Tools
+
+If you would like us to feature an API or tool that you’ve developed for use
+with our products and services, please send us an email at support@maxmind.com
```

Comment:
I haven't looked at any of the UAT feedback docs, but I *thought* we were moving toward using the https://support.maxmind.com/hc/en-us/requests/new form rather than emailing `support@maxmind.com` directly.

Reply:
https://github.com/maxmind/dev-site/pull/1199/commits/b6fa5c8304167e869443afe6d821647153212fd4